File: CastIt.Android/lib/models/dtos/empty_response_dto.dart
import 'package:json_annotation/json_annotation.dart';

part 'empty_response_dto.g.dart';

@JsonSerializable()
class EmptyResponseDto {
  @JsonKey(name: 'Succeed')
  bool succeed = false;

  @JsonKey(name: 'Message')
  String message;

  EmptyResponseDto({
    this.succeed,
    this.message,
  }) {
    succeed ??= false;
  }

  factory EmptyResponseDto.fromJson(Map<String, dynamic> json) =>
      _$EmptyResponseDtoFromJson(json);
}

File: CastIt.Domain/Dtos/EmptyResponseDto.cs
﻿namespace CastIt.Domain.Dtos
{
    public class EmptyResponseDto
    {
        public bool Succeed { get; set; }
        public string MessageId { get; set; }
        public string Message { get; set; }

        public EmptyResponseDto()
        {
        }

        public EmptyResponseDto(bool succeed, string message = null)
        {
            Succeed = succeed;
            Message = message;
        }
    }
}
